cargo-psp/src/main.rs

```diff
@@ -199,132 +200,148 @@ fn main() {
     };
 
     let cargo = env::var_os("CARGO").unwrap_or_else(|| "cargo".into());
-    let mut process = Command::new(cargo)
+    let mut build_process = Command::new(&cargo)
         .arg("build")
         .arg("-Z")
         .arg(build_std_flag)
         .arg("--target")
         .arg("mipsel-sony-psp")
+        .arg("--message-format=json-render-diagnostics")
         .args(args)
-        .stdin(Stdio::inherit())
-        .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit())
+        .stdout(Stdio::piped())
         .spawn()
         .unwrap();
 
-    let status = process.wait().unwrap();
+    let lone = {
+        let output = Command::new(cargo)
+            .arg("metadata")
+            .arg("--format-version=1")
+            .arg("--offline")
```

Comment:
this flag (--offline) is breaking something in our ci

Reply:
https://ci.mijalkovic.ca/teams/rust-psp/pipelines/rust-psp/jobs/run-tests-for-pr/builds/379.1#L62e61128:60

Reply:
deleted in 4e002f1